### PR TITLE
resolve gallery page

### PR DESCRIPTION
**Pull request by Mili940**

### CODE REVIEW

File: src/views/Gallery.vue [hunk 1]
```diff
@@ -1,13 +1,61 @@
...
+        class="col-md-3 pic"
+        @click="() => showImg(index)"
+      >
+        <img class="gallery__img" :src="src">
+      </div>
+    </div>
+    <vue-easy-lightbox
+      :visible="visible"
+      :imgs="imgs"
+      @hide="handleHide"
```
Comment (duka94) [inline src/views/Gallery.vue]: @hide="hideViewer()"

File: src/views/Gallery.vue [hunk 2]
```diff
@@ -1,13 +1,61 @@
...
+        visible: false,
+        index: 0   // default
+      }
+    },
+    methods: {
+      showImg (index) {
+        this.index = index
+        this.visible = true
+      },
+      handleHide () {
```
Comment (duka94) [inline src/views/Gallery.vue]: rename to hideViewer

File: src/views/Gallery.vue [hunk 3]
```diff
@@ -1,13 +1,61 @@
...
+        ],
+        // Img Url , string or Array
+        visible: false,
+        index: 0   // default
+      }
+    },
+    methods: {
+      showImg (index) {
+        this.index = index
+        this.visible = true
```
Comment (duka94) [inline src/views/Gallery.vue]: wrapp this.visible in new method, you can call that method
viewViewer() {
  this.viewer = true
}
and then in every line where you setting this.viewer = true just call this.viewViewer()

File: src/views/Gallery.vue [hunk 4]
```diff
@@ -1,13 +1,61 @@
...
+          '../../images/gallery/galerija9.jpeg',
+          '../../images/gallery/galerija3.jpeg',
+          '../../images/gallery/galerija4.jpeg',
+          '../../images/gallery/galerija5.jpeg',
+          '../../images/gallery/galerija6.jpeg',
+          '../../images/gallery/galerija7.jpeg',
+
+        ],
+        // Img Url , string or Array
+        visible: false,
```
Comment (duka94) [inline src/views/Gallery.vue:48]: rename to viewer

File: src/views/Gallery.vue [hunk 5]
```diff
@@ -1,13 +1,61 @@
...
+  <section class="gallery-page container">
+    <div class="gallery-page__header">
+      <h2 class="gallery-page__title">Galerija ordinacije Srećkov</h2>
     </div>
+    <div class="row">
+      <div
+        v-for="(src, index) in imgs"
+        :key="index"
+        class="col-md-3 pic"
+        @click="() => showImg(index)"
```
Comment (duka94) [inline src/views/Gallery.vue]: @click="showImg(index)"

File: src/views/Gallery.vue [hunk 6]
```diff
@@ -1,13 +1,61 @@
...
+      >
+        <img class="gallery__img" :src="src">
+      </div>
+    </div>
+    <vue-easy-lightbox
+      :visible="visible"
+      :imgs="imgs"
+      @hide="handleHide"
+    ></vue-easy-lightbox>
```
Comment (duka94) [inline src/views/Gallery.vue]: are you sure that this is correct by linter?

File: src/main.js [hunk 7]
```diff
@@ -2,9 +2,11 @@ import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
 import vueScrollto from 'vue-scrollto'
+import Lightbox from 'vue-easy-lightbox'
```
Comment (duka94) [inline src/main.js:5]: import LightBox just in Gallery view and delete from this file
image

File: src/assets/sass/gallery-page.scss [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+.gallery__img {
+  margin-bottom: 30px;
+  width: 100%;
+  cursor: pointer;
+
+  &:hover {
+    -webkit-box-shadow: 0 0 15px 2px rgba(0, 0, 0, .4);
+    box-shadow: 0 0 15px 2px rgba(0, 0, 0, .4);
+    -webkit-transition: all 0.4s;
```
Comment (duka94) [inline src/assets/sass/gallery-page.scss]: call transition mixin

**Review by duka94 — COMMENTED** [review]
fix js logic, if you have a question text me or call me

File: src/views/Gallery.vue (hunk 6 above)
Comment (Mili940) [reply src/views/Gallery.vue]: njet 💃

**Review by duka94 — APPROVED** [review]

**Pull request merged**